liveblog.php

```diff
@@ -34,13 +34,15 @@
 
 	/** Constants *************************************************************/
 
-	const version          = '1.4';
-	const rewrites_version = 1;
-	const min_wp_version   = '3.5';
-	const key              = 'liveblog';
-	const url_endpoint     = 'liveblog';
-	const edit_cap         = 'publish_posts';
-	const nonce_key        = 'liveblog_nonce';
+	const version                 = '1.4';
+	const rewrites_version        = 1;
+	const min_wp_version          = '3.5';
+	const min_wp_rest_api_version = '4.4';
+	const key                     = 'liveblog';
+	const url_endpoint            = 'liveblog';
+	const edit_cap                = 'publish_posts';
+	const nonce_key               = '_wpnonce'; // Using these strings since they're hard coded in the rest api. It'll still work fine for < 4.4
+	const nonce_action            = 'wp_rest';
```

Comment:
Right, the REST API always uses those values.
I'll make sure to double check the admin context.